### PR TITLE
Issue 50: Add volunteer section to home page

### PR DESCRIPTION
**Pull request by therealslimgrady**
https://github.com/nourimeals/website/issues/50

The home page had no section for visitors to volunteer. This PR adds a volunteer section using the same format as previous contact sections have.

Previously, the hero section had a small typo "as easy as easy." I changed this to say "is as easy."

Previous pull request came through forked copy, and requested changes were made from that PR (https://github.com/nourimeals/website/pull/53).

Current content at the bottom of the page: 
image

Content created with this pull request:
image

### CODE REVIEW

File: src/components/home/home.jsx [hunk 1]
```diff
@@ -80,6 +80,14 @@ class HomePage extends React.Component {
               </ul>
         </section>
 
+        <section id="home-volunteer">
+            <h2>Want to join the team?</h2>
+            <p>We welcome volunteers!</p>
+            <p>We work remotely and asynchronously using online collaboration tools like GitHub.<br />
+            Our stack is JavaScript/NodeJS, React, React-Native, and MongoDB.</p>
+            <button href="https://forms.gle/4ERxvyu5ThMC7dGU7" className="cta-btn">Volunteer</button>
```
Comment (ambergkim) [inline src/components/home/home.jsx]: Clicking the button does not work. Buttons need to be surrounded by "a" tags. Href attribute is not valid for buttons but is valid for "a" tags. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button
https://www.w3schools.com/tags/tag_a.asp

**Comment by ambergkim** [comment]
@therealslimgrady Just one thing to get the button working. Left a comment in the diff. :)

File: src/components/home/home.jsx (hunk 1 above)
Comment (therealslimgrady) [reply src/components/home/home.jsx]: Well, that's embarrassing. Not only did I know that, I didn't test the button.

Now that I have tested, the provided link doesn't work. It looks like the form is protected?

**Comment by ambergkim** [comment]
Merged: https://github.com/nourimeals/website/pull/56

**Pull request merged**